storage-resize-images/functions/src/config.ts

```diff
@@ -43,5 +43,5 @@ export default {
   includePathList: paramToArray(process.env.INCLUDE_PATH_LIST),
   excludePathList: paramToArray(process.env.EXCLUDE_PATH_LIST),
   deleteOriginalFile: deleteOriginalFile(process.env.DELETE_ORIGINAL_FILE),
-  imageType: process.env.IMAGE_TYPE,
+  imageTypes: process.env.IMAGE_TYPE,
```

Comment:
Can we make this an array here via `paramToArray` like includePathList & excludePathList do above - for consistency.

Reply:
Updated, testing below...

![image](https://user-images.githubusercontent.com/2060661/108851049-eaa68000-75db-11eb-8a43-a24b7ea885ef.png)